docs/protocol.md

```diff
@@ -326,142 +326,107 @@ a subsequent visit) some $passphrase$:
+| $`ct^{APKE} \gets \text{SD-APKE.AuthEnc}(sk_S^{APKE}, pk_{R,i}^{APKE}, pt, NR, -)`$ |                                 |                                                |
+| $`ct^{PKE} \gets \text{SD-PKE.Enc}(pk_{R,i}^{PKE}, pk_S^{APKE}, -, -)`$             |                                 |                                                |
+| $`C_S \gets (ct^{APKE}, ct^{PKE})`$                                                 |                                 |                                                |
+| $`x \gets^{\$} \mathcal{E}_H`$[^8]                                                  |                                 |                                                |
+| $`X \gets g^x`$                                                                     |                                 |                                                |
+| $`Z \gets (pk_{R,i}^{fetch})^x`$                                                    |                                 |                                                |
+|                                                                                     | $`\longrightarrow (C_S, X, Z)`$ |                                                |
+|                                                                                     |                                 | $`id \gets^{\$} \{0,1\}^{il}`$ for length $il$ |
+|                                                                                     |                                 | Store $(id, C_S, X, Z)$                        |
+
+### 7. Receiver fetches and decrypts messages <!-- Figure 2 as of 7703a58 -->
+
+A receiver knows their own keys.
+
+| All receivers  |
+| -------------- |
+| $pk_R^{APKE}$  |
+| $pk_R^{PKE}$   |
+| $pk_R^{fetch}$ |
+| $sk_R^{APKE}$  |
+| $sk_R^{PKE}$   |
+| $sk_R^{fetch}$ |
+
+For some newsroom $NR$:
+
+| Server                                                                                |                                                | Receiver                                                                        |
+| ------------------------------------------------------------------------------------- | ---------------------------------------------- | ------------------------------------------------------------------------------- |
+|                                                                                       |                                                | $`fetched \gets \emptyset`$                                                     |
+|                                                                                       | $\longleftarrow$ `RequestMessages`             |                                                                                 |
+| $`\forall C_i = (id_i, C_{S_i}, X_i, Z_i):`$ **TODO:** pad up to $n$ total challenges |                                                |                                                                                 |
```

Comment:
I'll either tack on a commit or open a follow-up pull request to fix this `TODO`, depending on the review timeline.